applications/nrf5340_audio/src/audio/sw_codec_select.c

```diff
@@ -196,8 +222,11 @@ int sw_codec_decode(uint8_t const *const encoded_data, size_t encoded_size, bool
 
 	static char pcm_data_stereo[PCM_NUM_BYTES_STEREO];
 
-	char decoded_data_mono[AUDIO_CH_NUM][PCM_NUM_BYTES_MONO] = {0};
-	char decoded_data_mono_system_sample_rate[AUDIO_CH_NUM][PCM_NUM_BYTES_MONO] = {0};
+	char decoded_data_stereo[PCM_NUM_BYTES_MONO * m_config.decoder.num_ch];
+	char decoded_data_stereo_system_sample_rate[PCM_NUM_BYTES_MONO * m_config.decoder.num_ch];
+
+	char *dec_data_stereo;
+	char *dec_data_stereo_src;
```

Comment:
I guess src here is meant to be Sample_Rate_Converted? `src` is easily confues as source, so would recommend something else